### PR TITLE
fix: Replace existing usages of TextInputFieldV2 with TextFormField

### LINKED ISSUE #4316: Replace existing usages of `TextInputFieldV2` with `TextFormField`
**Describe the (not a bug)**
Currently we have `TextFormField` and `TextInputFieldV2` tailwind component to achieve the same.
To improve maintainability and reduce duplication, change all existing usages of `TextInputFieldV2` to `TextFormField` and delete the `TextInputFieldV2` component.

Blocked as https://github.com/coronasafe/care_fe/pull/4309 waiting to be merged.

### PR DESCRIPTION
**Pull request by cp-Coder**
# Bug Fix

## Proposed Changes

- Replaced `InputFieldV2` and `InputTextField` with `TextFormField`
- Modified `BedSelect` component

## Associated Issue

- Fixes #4316

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug/test a new feature.
- [ ] Update product [documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care)
- [ ] Ensure that UI text is kept in I18n files
- [ ] Prep the screenshot or demo video for the changelog entry, and attach it to issue
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/Consultations/Beds.tsx [hunk 1]
```diff
@@ -124,44 +129,38 @@ const Beds = (props: BedsProps) => {
...
+                error=""
                 multiple={false}
-                margin="dense"
                 facility={facilityId}
               />
             </div>
             <div>
-              <InputLabel htmlFor="date_declared_positive">
+              <FieldLabel htmlFor="date_declared_positive">
                 Date of Shift
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/Beds.tsx]:
```suggestion
```

File: src/Components/Facility/Consultations/Beds.tsx [hunk 2]
```diff
@@ -124,44 +129,38 @@ const Beds = (props: BedsProps) => {
...
-              </InputLabel>
-              <TextInputField
-                name="date_declared_positive"
+              </FieldLabel>
+              <DateFormField
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/Beds.tsx]:
```suggestion
              <DateFormField
                label="Date of Shift"
```

File: src/Components/Assets/configure/CameraConfigure.tsx [hunk 3]
```diff
@@ -34,33 +34,32 @@ export default function CameraConfigure(props: CameraConfigureProps) {
   return (
     <Fragment>
       <Card>
-        <form onSubmit={addPreset}>
+        <form onSubmit={addPreset} className="">
```
Comment (rithviknishad) [inline src/Components/Assets/configure/CameraConfigure.tsx:37]:
```suggestion
        <form onSubmit={addPreset}>
```

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Comment by khavinshankar** [comment]
@cp-Coder can you take off the failing cypress tests, most likely merging the latest develop branch should fix it

**Comment by khavinshankar** [comment]
@cp-Coder checkout the failing cypress tests

**Comment by nihal467** [comment]
LGTM

**Pull request merged**